ex4/index.js

```diff
@@ -0,0 +1,6 @@
+const addNumber = (...argumente) => argumente.reduce((suma, numar) => suma + numar, 0);
```

Comment:
```suggestion
const getSum = (...argumente) => argumente.reduce((suma, numar) => suma + numar, 0);
```
numele functiei nu este chiar sugestim. "addNumber"?
Ce face functia? Returneaza suma tuturor argumentelor. Un nume mai potrivit ar fi "getSum" sau ceva de genul, ce zici?